scripts/version.js

```diff
@@ -0,0 +1,154 @@
+const shell = require("shelljs");
+const argv = require("yargs").argv;
+
+if (argv.type) {
+  switch (argv.type) {
+    case "make-stable": {
+      if (argv.fromVersion && argv.toVersion) {
+        const silentState = shell.config.silent; // save old silent state
+        shell.config.silent = true;
+        const fromVersion = argv.fromVersion;
+        const toVersion = argv.toVersion;
+
+        // configuration-api
```

Comment:
we would need support for Chat Widget (extending-chat-widget category). they have different versions and don't have rtm-reference/